bigbone/src/main/kotlin/social/bigbone/api/entity/WebPushSubscription.kt

```diff
@@ -0,0 +1,94 @@
+package social.bigbone.api.entity
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * Represents a subscription to the push streaming server.
+ * @see <a href="https://docs.joinmastodon.org/entities/WebPushSubscription/">Mastodon API Push</a>
+ */
+@Serializable
+data class WebPushSubscription(
+
+    /**
+     * The ID of the Web Push subscription in the database.
+     */
+    @SerialName("id")
+    val id: String = "",
+
+    /**
+     * Where push alerts will be sent to.
+     */
+    @SerialName("endpoint")
+    val endpoint: String = "",
+
+    /**
+     * The streaming server’s VAPID key.
+     */
+    @SerialName("server_key")
+    val serverKey: String = "",
+
+    /**
+     * Which alerts should be delivered to the endpoint.
+     */
+    @SerialName("alerts")
+    val alerts: Alerts
+)
+
+/**
+ * Which alerts should be delivered to the endpoint.
+ * @see <a href="https://docs.joinmastodon.org/entities/WebPushSubscription/">Mastodon API Push</a>
+ */
+@Serializable
+data class Alerts(
+    /**
+     * Receive mention notifications?.
```

Comment:
For property documentation we usually use the official description in the Mastodon API docs. In this case, it says `Receive a push notification when someone else has mentioned you in a status?`. Would be cool if you could replace the KDoc for the other properties too. Thank you!